cli_pr_review.py

```diff
@@ -11,7 +11,7 @@
 app = typer.Typer()
```

Comment:
The updated spinner text in line 11 seems more appropriate for this pull request. Good change!